src/scan-imports.ts

```diff
@@ -48,17 +48,26 @@ function removeSpecifierQueryString(specifier: string) {
   return specifier;
 }
 
+function removeSpecifierQuotes(specifier: string) {
+  const strippedSpecifier = specifier.replace(/[\'\"]/g, '');
```

Comment:
nit: `return specifier.replace(/[\'\"]/g, '');`

Reply:
Actually: for fun I did a quick jsperf of `replace()` vs. `substring` and the results were actually pretty significant.

<img width="997" alt="Screen Shot 2020-02-16 at 10 01 32 AM" src="https://user-images.githubusercontent.com/622227/74609897-728dac00-50a3-11ea-992a-068fc49d4f8b.png">

So we may want to do something more like:

```js
function getSpecifierFromImport(code, imp) {
  // TODO: Add comment explaining why
  if (imp.d) {
    return code.substring(start + 1, end - 1);
  }
  return code.substring(start, end);
}
```

Bonus: then you're always passing a clean specifier to `parseWebModuleSpecifier()`

Reply:
Nice! Definitely going to use `substring` then